webserver/style.css

```diff
@@ -678,3 +687,29 @@ button:hover {
 .hidden {
   display: none;
 }
+
+input.valid {
+  border-color: #468847;
```

Comment:
Utilize CSS variables to ensure consistency and to maintain various color schemes for theming.
https://github.com/melkati/CO2-Gadget/blob/development/webserver/style.css#L1-L65